Dockerfile

```diff
@@ -5,9 +5,12 @@ COPY / .
 RUN make get \
 && make build
 
-FROM chromedp/headless-shell:latest
+FROM chromedp/headless-shell:stable
 RUN apt-get update \
-&& apt-get install --no-install-recommends -qq ca-certificates bash sed git dumb-init \
```

Comment:
I get the point of pinning it, unfortunately stable and latest are moving targets, so I would like to avoid that it suddenly breaks.

Reply:
Sorry about this. This was never intended to be pushed in this repo. It is an internal constraint on our side.